homie-influx/mappings.example.toml

```diff
@@ -0,0 +1,3 @@
+[[mapping]]
```

Comment:
`[[mappings]]` is probably more what you mean. It's just chance that toml's array of tables syntax is a bit mental

If someone spelt it like this then it is clear that plural is correct:
```
mappings = [
    {homie_prefix="homie", influxdb_database="test"},
    {homie_prefix="homie2", influxdb_database="test2"},
]
```

(spoilers, I think toml is a bit mental)

Reply:
Done.